backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/callout/C_Order.java

```diff
@@ -153,4 +154,18 @@ public void updateHandOverAddressForceUpdateCaptureLocation(final I_C_Order orde
 		documentLocationBL.updateCapturedLocation(OrderDocumentLocationAdapterFactory.handOverLocationAdapter(order));
 	}
 
+	@CalloutMethod(columnNames = {
+			I_C_Order.COLUMNNAME_M_Shipper_ID },
+			skipIfCopying = true)
+	public void updateDeliveryViaRule(final I_C_Order order)
+	{
+		if(order.getM_Shipper() != null)
```

Comment:
```suggestion
		if(order.getM_Shipper_ID() > 0)
```

no need to try and load the whole thing